### PR TITLE
Oci note

### PR DESCRIPTION
**Pull request by SharpRake**
## Type of change

This is a minor update to [How to Connect Chainguard Enforce to Private Container Registries](https://edu.chainguard.dev/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/connecting-to-private-registries/). The goal is to add a clarifying point about what values are accepted by the `oci` key in CIPs.

Also, note that when I cloned the repo this doc had a lot of unnecessary line breaks throughout. I also removed those to ensure that it renders properly. 

### What should this PR do?

This PR fulfills the request for documentation within [this issue](https://github.com/chainguard-dev/customer-issues/issues/68) (now closed)

### CODE REVIEW

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/connecting-to-private-registries.md [hunk 1]
```diff
@@ -21,21 +21,15 @@ Chainguard Enforce supports two methods of authenticating to private registries:
...
-applies to all the images scoped to that policy. 
+To support the widest set of private registries, you can authenticate using a Kubernetes image pull secret. This is configured on a per-policy basis, and applies to all the images scoped to that policy. 
 
 ### Configuring your image pull secret
 
-To verify policies using attestation and signatures for images in a private
-registry, create an image pull secret in the `cosign-system` namespace of your
-cluster, filling in your own details for the private registry, username,
-password, and email:
+To verify policies using attestation and signatures for images in a private registry, create an image pull secret in the `cosign-system` namespace of your cluster, filling in your own details for the private registry, username, password, and email:
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/connecting-to-private-registries.md:32]:
```suggestion
To verify policies using attestation and signatures for images in a private registry, create an image pull secret in the `cosign-system` namespace of your cluster. Here, the `docker-registry` subcommand creates a secret that is specifically for use with a Docker registry. We are initializing this secret with the `regcreds` variable, which will hold the secret's value.

In the next lines, fill in your own details for the private registry, username, password, and email, substituting those values for the variables.

```

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/connecting-to-private-registries.md [hunk 2]
```diff
@@ -80,17 +68,11 @@ spec:
       - name: regcreds
 ```
 
-In this example, add the name of your private registry directory in `glob` and
-include the name of the private registry in `oci`. You will have set up
-`regcreds` in the previous `kubectl create secret` command. 
+In this example, add the name of your private registry directory in `glob` and include the name of the private registry in `oci`. Note that the value you pass to the `oci` key cannot be a glob pattern and must be a valid repository name. Also, be aware that you will have set up `regcreds` in the previous `kubectl create secret` command.
```
Comment (ltagliaferri) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/connecting-to-private-registries.md:71]:
```suggestion
In this example, add the name of your private registry directory in `glob` and include the name of the private registry in `oci`. Note that the value you pass to the `oci` key cannot be a glob pattern and must be a valid repository name. Also, be aware that you will have set up `regcreds` in the previous `kubectl create secret` command; this is the variable you initialized to hold the relevant `docker-registry` subcommand secret.

```

**Review by ltagliaferri — APPROVED** [review]
On second look at this, I think we can add a little context on the `regcreds` variable. Let me know if you think I added enough context, and feel free to add more or modify as you see fit.

Otherwise this looks good!

File: content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/connecting-to-private-registries.md (hunk 1 above)
Comment (SharpRake) [inline content/chainguard/chainguard-enforce/chainguard-enforce-kubernetes/connecting-to-private-registries.md:32]:
```suggestion
To verify policies using attestation and signatures for images in a private registry, create an image pull secret in the `cosign-system` namespace of your cluster. The following example creates a secret named `regcreds`, but you could provide a different name if you like. Additionally, the `docker-registry` subcommand creates a secret that is specifically for use with a Docker registry.

In the next lines, fill in your own details for the private registry, username, password, and email, substituting those values for the variables.
```

**Pull request merged**